app/controllers/api/v1/employees_controller.rb

```diff
@@ -3,16 +3,10 @@
 module Api
   module V1
     # Employees API controller
-    class EmployeesController < ApplicationController
-      before_action :authenticate_user!
-      rescue_from ActiveRecord::RecordNotFound, with: :record_not_found
-      rescue_from ActiveRecord::RecordInvalid, with: :record_invalid
-      rescue_from ActionPolicy::Unauthorized, with: :handle_unauthorized
-      rescue_from ArgumentError, with: :params_invalid
-
+    class EmployeesController < UserController
```

Comment:
I don't think that it's a good idea to inherit from contoller, just use modules if needed

Reply:
my aim was following DRY principle
I don't want repeat this code in each controller:
```
      before_action :authenticate_user!
      rescue_from ActiveRecord::RecordNotFound, with: :record_not_found
      rescue_from ActiveRecord::RecordInvalid, with: :record_invalid
      rescue_from ActionPolicy::Unauthorized, with: :handle_unauthorized
      rescue_from ArgumentError, with: :params_invalid
```
I have seen this method few times, for example: https://rmulhol.github.io/ruby/2015/04/27/solid-rails.html

Reply:
yeah, but you inherit from UserController, use Application instead